core/deployment/src/main/java/io/quarkus/deployment/mutability/ReaugmentTask.java

```diff
@@ -1,7 +1,6 @@
 package io.quarkus.deployment.mutability;
 
-import static io.quarkus.deployment.pkg.steps.JarResultBuildStep.BUILD_SYSTEM_PROPERTIES;
-import static io.quarkus.deployment.pkg.steps.JarResultBuildStep.DEPLOYMENT_QUARKUS;
+import static io.quarkus.deployment.pkg.steps.JarResultBuildStep.*;
```

Comment:
We probably don't want this

Reply:
oops, I am using a different computer and the IDE was not fully setup properly.

Reply:
Seems like your IDE is still messing with you :)